k8s/admin/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```
@@ -187,6 +187,12 @@ spec:
                     name: RD
                     recurse: true
                     title: Ref Data Dashboard
+                - buildMonitor:
+                    includeRegex: >-
+                      ^HMCTS_.*ADOPTION.*\/.+\/master
```

mirroring https://github.com/hmcts/cnp-jenkins-config/pull/403#discussion_r381335673
```suggestion
                      ^HMCTS_.*Adoption.*\/.+\/master
```

---

thanks @timja. Now updated